mesh_map/src/util.cpp

```diff
@@ -234,8 +234,13 @@ void getRainbowColor(float value, float& r, float& g, float& b)
 
 mesh_map::Normal poseToDirectionVector(const geometry_msgs::msg::PoseStamped& pose, const tf2::Vector3& axis)
 {
-  tf2::Stamped<tf2::Transform> transform;
-  fromMsg(pose, transform);
```

Comment:
One issues I'd like to highlight is that `tf2_geometry_msgs` does not seem to provide a conversion between `geometry_msgs::msg::PoseStamped pose` and `tf2::Stamped<tf2::Transform> transform`. Or between `Pose` and `Transform`.
So this is my workaround for now, I hope the resulting direction vector remains the same.